kolibri/deployment/default/urls.py

```diff
@@ -24,7 +24,7 @@
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^$', include('kolibri.core.urls')),
+    url(r'^kolibri/', include('kolibri.core.urls')),
     url(r'', include('kolibri.content.urls')),
     url(r'^api/', include('kolibri.auth.urls')),
     url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
```

Comment:
On the API front, I have an outstanding card for this:
https://trello.com/c/iQ2nF7K9/89-api-urls-are-namespaced-by-app-with-an-api-root
(please add comments there, or we can follow up once the URL stuff in this PR is merged)